lms/views/api/vitalsource.py

```diff
@@ -0,0 +1,37 @@
+from pyramid.view import view_config, view_defaults
+
+from lms.security import Permissions
+from lms.services.exceptions import ProxyAPIError
+from lms.views.api.vitalsource_sample_data import book_data, toc_data
+
+
+class BookNotFoundError(ProxyAPIError):
+    def __init__(self, book_id):
+        super().__init__(f"Book {book_id} not found")
```

Comment:
This should be in `views/api/exceptions.py` (which is where we keep exception classes). Alternatively you can probably get away with just raising `ProxyAPIError` directly without subclassing it

Reply:
> This should be in views/api/exceptions.py (which is where we keep exception classes)

Ah, OK. `exceptions.py` defines exception views but all the exception subclasses that I can see in it are re-exported from elsewhere, typically a relevant service.

I have implemented your second suggestion of just using `ProxyAPIError` directly for now, since there is no added functionality in the custom subclass and nothing relies on its instance type being different than `ProxyAPIError`.